### PR TITLE
corrections and updates

### PR DESCRIPTION
**Pull request by thea-m**
(@eu-genia , could you delete the reference to the discussions (and possibly substitute with issues or e-mail, whatever you think best) page on https://betamasaheft.eu/help.html?)

### CODE REVIEW

**Comment by eu-genia** [comment]
Please check https://betamasaheft.eu/help.html now

**Comment by eu-genia** [comment]
https://github.com/BetaMasaheft/BetMas/commit/43ac7454c45d253efae8f8e6d6e079e5ee68124a

**Comment by thea-m** [comment]
Great, thank you!

File: pages/images.xml [hunk 1]
```diff
@@ -106,7 +107,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </msIdentifier>
   </egXML>
 <p>To add the value for the <att>facs</att> of an image or range to be linked you can check the number in the viewer (after the above step has been saved and the images appear).
-The sequence number of the images is the one you want to assign. in <ref target="https://betamasaheft.eu/manuscripts/BMLacq755/viewer"/> you can see in the sequence of
+   The sequence number of the images is the one you want to assign. in <ref target="https://betamasaheft.eu/manuscripts/BMLacq755/viewer">BMLacq755</ref> you can see in the sequence of
```
Comment (DenisNosnitsin1970) [inline pages/images.xml:110]: you want to assign. In

**Pull request merged**